api/types/types.go

```diff
@@ -75,6 +75,10 @@ type ImageInspect struct {
 
 	// ContainerConfig is the configuration of the container that was committed
 	// into the image.
+	//
+	// Depending on how the image was created, ContainerConfig, or its fields
```

Comment:
For the `ContainerConfig` field inside image I would say:

ContainerConfig is an optional field containing the configuration of the container that was last committed when creating the image. Previous versions of Docker builder used this field to store build cache and it is not in active use anymore.